data/gsettings/com.gexperts.Tilix.gschema.xml

```diff
@@ -749,6 +760,10 @@
       <default>'narrow'</default>
       <summary>Whether ambiguous-width characters are narrow or wide when using UTF-8 encoding</summary>
     </key>
+    <key name="ctrl-backspace" enum="com.gexperts.Tilix.EraseControl">
+      <default>'none'</default>
+      <summary>Makes the terminal threat "Control+Backspace" as different control instead of "Backspace". May be used as a shortcut for removing deleting a word instead of a character.</summary>
```

Comment:
"threat" should be a typo of "treat". "removing deleting" should also be a typo.